From c7eaf9865889d4b46646320afe262ef22d5c96f9 Mon Sep 17 00:00:00 2001
From: Holger Freyther <ich@holger-freythers-macbook-pro.local>
Date: Mon, 19 Jan 2009 01:54:37 +0100
Subject: [PATCH] Ignore the ambox information of articles...

they talk about quality but also about pending cleanups and may
lead to discussion pages...
---
 skins/monobook/main.css |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/skins/monobook/main.css b/skins/monobook/main.css
index 00b2293..849f1a5 100644
--- a/skins/monobook/main.css
+++ b/skins/monobook/main.css
@@ -1559,6 +1559,7 @@ div.nounderlines,
 div.infobox,
 table.collapsible,
 table.gallery,
+table.ambox,
 span#coordinates {
     display: none;
 }
-- 
1.6.0.2

